src/content/docs/security/security-privacy/compliance/regulatory-audits-new-relic-services.mdx

```diff
@@ -241,7 +241,7 @@ New Relic's time frames for supported regulatory frameworks and annual audits in
 
     <tr>
       <td>
-        Mobile monitoring
+        [Mobile monitoring](/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#mobile-endpoints)
```

Comment:
We're not consistent here because we use "Mobile monitoring" in this document but "Mobile agent" over in the endpoints doc. Maybe change to "Mobile monitoring agents"? Does this matter?

Reply:
it probably doesn't matter but I changed to align to 'mobile agents'